src/world/partner/kooper.c

```diff
@@ -71,6 +71,14 @@ ApiStatus func_802BD228_31B248(Evt* script, s32 isInitialCall) {
     return partner_get_out(kooper) ? ApiStatus_DONE1 : ApiStatus_BLOCK;
 }
 
+EvtSource world_kooper_take_out = {
+    EVT_CALL(func_802BD228_31B248)
+    EVT_RETURN
+    EVT_END
+};
+
+unkPartnerStruct* D_802BEB60_31CB80 = 0x802BEC80;
```

Comment:
this symbol may be bss. you can find out by seeing if it goes past the end of the "natural" overlay length (get the rom size of the overlay and add the ram start to the size to get the ram end. if the symbol is past that, it's either bss or from a different overlay)

Reply:
I see there's 1 or 2 more of these. the solution would be to add them to undefined_syms and then extern a new symbol for them